astropy/io/ascii/tests/test_write.py

```diff
@@ -850,9 +849,10 @@ def test_write_overwrite_ascii(
         assert not os.path.exists(filename)
 
 
-fmt_name_classes = list(
-    chain(ascii.core.FAST_CLASSES.items(), ascii.core.FORMAT_CLASSES.items())
-)
+fmt_name_classes = [
+    *ascii.core.FAST_CLASSES.items(),
+    *ascii.core.FORMAT_CLASSES.items(),
+]
```

Comment:
Single line?

Reply:
Black has spoken?

Reply:
It may just be that removing the trailing comma on the second item, it speaks differently (sorry, too lazy to count to see if it fits on a line...)

Reply:
89😆 I'd be happy to rename this to `fmt_name_cls` as it is only used twice below, but can also keep it as is then.

Reply:
Fine to keep as is!

Reply:
For some reason I can't claim to understand Black insists having the iterator unpackings on separate lines. Run the following in a shell and see for yourself:
```shell
black --diff --code 'fmt_name_classes = [
    *ascii.core.FAST_CLASSES.items(), *ascii.core.FORMAT_CLASSES.items()
]'
```
I don't think the formatting here is bad enough that fighting Black would be worth the effort.